Doc/Tutorial/chapter_entrez.tex

```diff
@@ -511,15 +511,16 @@ \section{ELink: Searching for related items in NCBI Entrez}
 %cont-doctest
 \begin{minted}{pycon}
 >>> len(record[0]["LinkSetDb"])
-7
+8
 \end{minted}
 \noindent The exact numbers should increase over time:
 \begin{minted}{pycon}
```

Comment:
Does this example make sense to include as a continued doctest? I suspect it will be too fragile (changing results over time).